plugins/glustershd/rest.go

```diff
@@ -63,6 +64,73 @@ func getHealInfo(volname string, option string) (string, error) {
 	return runGlfshealBin(volname, options)
 }
 
+func filterHealInfo(healInfo glustershdapi.HealInfo) (glustershdapi.HealInfo, error) {
+	var err error
+	for i := range healInfo.Bricks {
+		if healInfo.Bricks[i].TotalEntriesRaw != "" {
+			healInfo.Bricks[i].TotalEntries = new(int64)
+			if healInfo.Bricks[i].TotalEntriesRaw == "-" {
+				*healInfo.Bricks[i].TotalEntries = -1
+			} else {
+				*healInfo.Bricks[i].TotalEntries, err = strconv.ParseInt(healInfo.Bricks[i].TotalEntriesRaw, 10, 64)
+				if err != nil {
+					return healInfo, err
+				}
+			}
+		}
+
+		if healInfo.Bricks[i].EntriesInHealPendingRaw != "" {
```

Comment:
create a function to avoid duplicates.

```
func convertToNumber(value string) int64 {

    if value == "" {
        return 0
    }

    if value == "-" {
        return -1
    }
    val, err := strconv.ParseInt(value, 10, 64)
    if err != nil {
        return -1
    }
    return val
}

Reply:
Done